pkg/docker/cockpit-atomic-storage

```diff
@@ -428,8 +424,10 @@ def cmd_monitor():
         if len(r) > 0:
             if mon.stdout.readline().decode().startswith("UDEV  "):
                 got_event = True
+        # we want to periodically all get_info() to update the usage
```

Comment:
I think there might be a typo here... if there's a correct phrase, I can update during merge